macros/audit_logging/execution.sql

```diff
@@ -19,7 +19,7 @@
 
     create table if not exists {{ pageup_dbt_utils.get_execution_relation() }}
     (
-        execution_id        uuid PRIMARY KEY NOT NULL,
+        execution_id        varchar(250) PRIMARY KEY NOT NULL,
```

Comment:
wouldnt 40 be better?

Reply:
just using 250 since we changed rdl to use 250